Connection.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Data;
+using System.Configuration;
+using FirebirdSql.Data.FirebirdClient;
+
+namespace ProjetoDesafio
+{
+    public class Connection
+    {
+        private static readonly Connection InstanciaFireBird = new Connection();
+        private Connection(){}
+
+        public static Connection GetInstancia()
+        {
+            return InstanciaFireBird;
+        }
```

Comment:
Aqui dá pra transformar naqueles `=>`, sabe?
```suggestion
        public static Connection GetInstancia() =>
            InstanciaFireBird;
```